### PR TITLE
GLT-1151 final simplifying of side menu

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

**Comment by morgantaschuk** [comment]
I notice that the page title bar still says 'List Samples' in the version on production.
screen shot 2016-09-21 at 12 25 59

**Comment by callunity** [comment]
Aha, I hadn't realized that the page titles were in scope. I will change them.

File: miso-web/src/main/webapp/pages/adminsub.jsp [hunk 1]
```diff
@@ -50,6 +50,11 @@
         <li><a href="<c:url value="/miso/indices"/>">Indices</a></li>
         <li><a href="<c:url value="/miso/experiments"/>">Experiments</a></li>
         <li><a href="<c:url value="/miso/studies"/>">Studies</a></li>
+      <sec:authorize access="hasRole('ROLE_ADMIN')">
+        <br/>
+        <li><a href="<c:url value="/miso/admin/users"/>">Users</a></li>
+        <li><a href="<c:url value="/miso/admin/groups"/>">Groups</a></li>
```
Comment (djcooke) [inline miso-web/src/main/webapp/pages/adminsub.jsp]: Users and Groups should probably still be under a separate 'User Administration' heading

Comment (callunity) [inline miso-web/src/main/webapp/pages/adminsub.jsp]: Fixed.

**Pull request merged**